conf/messages.cy

```diff
@@ -1,5 +1,8 @@
 service.name = Datgan nwyddau masnachol sy’n cael eu cario mewn bagiau neu gerbydau bach
 service.homePageUrl = /declare-commercial-goods
+service.name.import.a.href = https://www.gov.uk/guidance/declare-commercial-goods-youre-bringing-into-great-britain-in-your-accompanied-baggage-or-small-vehicles
+service.name.export.a.href = https://www.gov.uk/guidance/declare-commercial-goods-youre-taking-out-of-great-britain-in-your-accompanied-baggage-or-small-vehicles
+service.name.none.a.href = https://www.gov.uk/
```

Comment:
not a biggy, but instead of `none` , using `default` or `govuk` makes more sense

Reply:
changed to `govuk`